java/src/main/java/com/powsybl/dataframe/network/extensions/AbstractSingleDataframeNetworkExtension.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2021-2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.dataframe.network.extensions;
+
+import com.powsybl.dataframe.network.NetworkDataframeMapper;
+
+import java.util.*;
+
+/**
+ * @author Hugo Kulesza <hugo.kulesza@rte-france.com>
+ */
+public abstract class AbstractSingleDataframeNetworkExtension implements NetworkExtensionDataframeProvider {
+
+    public List<Optional<String>> getExtensionTableNames() {
```

Comment:
In collection classes too it's a little unusual to use optionals, I would prefer plain strings instead. Actually in this list in particular, I think we should not have null values at all.
(Note that it's possible that some map implementations don't support null key, but I think for ex. `HashMap` does)